### PR TITLE
gcc@8 (new formula)

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: Formula/gcc@8.rb [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      #   * `-L#{libdir}` instructs gcc to find the corresponding gcc
+      #     libraries. It is essential if there are multiple brewed gcc
+      #     with different versions installed.
+      #     Noted that it should only be passed for the `gcc@*` formulae.
+      #   * `-L#{HOMEBREW_PREFIX}/lib` instructs gcc to find the rest
+      #     brew libraries.
+      libdir = HOMEBREW_PREFIX/"lib/gcc/#{version_suffix}"
+      specs.write specs_string + <<~EOS
+        *cpp_unique_options:
+        + -isysroot /nonexistent #{system_header_dirs.map { |p| "-idirafter #{p}" }.join(" ")}
```
Comment (sjackman) [inline Formula/gcc@8.rb]: Can you please change this line as per https://github.com/Linuxbrew/homebrew-core/issues/7461#issuecomment-386374091 ?

**Comment by sjackman** [comment]
Thanks for this PR, @xu-cheng ! One minor change. See https://github.com/Linuxbrew/homebrew-core/issues/7461#issuecomment-386374091

**Comment by iMichka** [comment]
@xu-cheng I had no time to look at this. l hope I can review this in detail tomorrow.

**Comment by iMichka** [comment]
@xu-cheng Thanks! Great work here.

If you have time, could you open PR's to fix the remaining `nonexistent` paths in the other gcc formulae? :)

**Comment by sjackman** [comment]
Thanks, @xu-cheng !

**Pull request closed without merging**